modules/plugins/__main__.py

```diff
@@ -1,31 +1,42 @@
 import ez
 
-from . import router
+import ez.log
 
-del router
+ez.log.info("Plugin Manager loaded")
+
+
+# from . import router
+
+# del router
```

Comment:
Another

Reply:
Ok, yeah.

I'll go through the comments and update you when I'm done.